docs/source/technical_tutorials/access/direct_access.ipynb

```diff
@@ -0,0 +1,1607 @@
+{
```

Comment:
<p>specify "Earth Data Cloud" datasets which are stored in AWS S3.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/344/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>